projects/aca-content/src/lib/components/details/details.component.spec.ts

```diff
@@ -140,4 +181,36 @@ describe('DetailsComponent', () => {
     fixture.detectChanges();
     expect(store.dispatch).toHaveBeenCalledWith(new SetSelectedNodesAction([node]));
   });
+
+  it('should set aspectActions from extensions', () => {
+    extensionsServiceMock.getAllowedSidebarActions.and.returnValue(of(mockAspectActions));
+    fixture.detectChanges();
+    fixture.whenStable().then(() => {
```

Comment:
Are you sure this test is correct? You use aync `whenStable` without async or fakeAsync

Reply:
Updated

Reply:
Done